kafka-ui-react-app/src/components/Topics/List/ThroughputCell.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { CellContext } from '@tanstack/react-table';
+import { Topic } from 'generated-sources';
+import BytesFormatted from 'components/common/BytesFormatted/BytesFormatted';
+
+function formatThroughput(row: Topic) {
```

Comment:
why are we creating extra function reference in the memory and we only use it in a single place , i think we can move this logic to the component.